virtcontainers/utils/utils.go

```diff
@@ -234,9 +234,6 @@ func BuildSocketPath(elements ...string) (string, error) {
 
 // SupportsVsocks returns true if vsocks are supported, otherwise false
 func SupportsVsocks() bool {
-	if _, err := os.Stat(VSockDevicePath); err != nil {
```

Comment:
It looks like you can remove `VSockDevicePath` entirely now as it's only being used by the tests?

Reply:
yes, already updated. ;)